mmtrack/models/mot/strong_sort.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Dict, Optional
+
+from torch import Tensor
+
+from mmtrack.utils import OptConfigType, SampleList
+from mmtrack.registry import MODELS, TASK_UTILS
+from .deep_sort import DeepSORT
+
+
+@MODELS.register_module()
+class StrongSORT(DeepSORT):
+    """StrongSORT: Make DeepSORT Great Again.
+
+    Details can be found at `StrongSORT<https://arxiv.org/abs/2202.13514>`_.
+
+    Args:
+        detector (dict): Configuration of detector. Defaults to None.
+        reid (dict): Configuration of reid. Defaults to None
+        tracker (dict): Configuration of tracker. Defaults to None.
+        kalman (dict): Configuration of Kalman filter. Defaults to None.
+        cmc (dict): Configuration of camera model compensation.
+            Defaults to None.
+        data_preprocessor (dict or ConfigDict, optional): The pre-process
+           config of :class:`TrackDataPreprocessor`.  it usually includes,
+            ``pad_size_divisor``, ``pad_value``, ``mean`` and ``std``.
+        init_cfg (dict or list[dict]): Configuration of initialization.
+            Defaults to None.
+    """
+
+    def __init__(self,
+                 detector: Optional[dict] = None,
+                 reid: Optional[dict] = None,
+                 tracker: Optional[dict] = None,
+                 kalman: Optional[dict] = None,
+                 cmc: Optional[dict] = None,
+                 data_preprocessor: OptConfigType = None,
+                 init_cfg: OptConfigType = None):
+        super().__init__(data_preprocessor=data_preprocessor,
+                         init_cfg=init_cfg)
```

Comment:
if you want to inherit from `DeepSORT`, you should also pass `detector`, `tracker` and so on into `__init__`, since they will be build in the `__init__`